tests/system/Validation/FormatRulesTest.php

```diff
@@ -80,6 +80,45 @@ public function testRegexMatchFalse(): void
         $this->assertFalse($this->validation->run($data));
     }
 
+    public function testValidJWTValid(): void
```

Comment:
Thank you for including tests! I think there are a lot more "edge cases" to check. I would like to see these tests refactored with a data provider. We have lots of examples you could mimic in here already, but basically it would be an array of `[string $tokenToTest, bool $expectedOutcome]`. This way you can test lots of input in one pass and it is easy to see what cases are covered.